tests/foreman/cli/test_organization.py

```diff
@@ -1693,3 +1694,33 @@ def test_positive_info_by_name(self):
         org = make_org()
         result = Org.info({'name': org['name']})
         self.assertEqual(org['id'], result['id'])
+
+    @tier1
+    def test_positive_multibyte_latin1_org_names(self):
+        """Hammer Multibyte and Latin-1 Org names break list pagination
+
+        :id: 4fa0afe7-6d0a-4c3e-a0fc-4ecb95c50fc9
+
+        :BZ: 1418412
```

Comment:
No, I think somehow this was carried over from my last pull request. https://github.com/SatelliteQE/robottelo/pull/4896